### DISCUSSION TITLE
RFM 95w Transmit mode not get set to value 0x83 in operating mode Lora  mode

### DISCUSSION THREAD

**Discussion by vaibhavdawalkar** [Lora-net/LoRaMac-node#1242] [Q&A]
Respected ,

I have two module RFM95w . One code for transmit and other code is receiving only for operation Lora mode 0x80.
code work fine but when the module which i used as transmit mode then use in receiving mode it working fine.
But module which on previously use in receiving mode and now use in transmit mode its transmit mode not get set it remain in 0x09 or 0x03 not set to 0x83.
Waiting for valuable reply.

**Comment by mluis1** [comment]
Unfortunately, you do not provide enough details in order for someone to be able to help you.

The only thing that I can say from your description is that for some reason the radio configuration has changed from LoRa modulation to FSK modulation.

If you could provide a code example which reproduces the issue it would be more helpful as one could try to run it and figure out what is wrong.

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.

**Reply by vaibhavdawalkar** [reply]
Thanks for valuable reply.
We facing issue of switching lora transmit mode from lora receive mode.
Please get attached files,
